Tests/Tests.xcodeproj/project.pbxproj

```diff
@@ -429,13 +429,19 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 435F03B32174AB9600A86B2D /* Module-Debug.xcconfig */;
 			buildSettings = {
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Well, but why do we still use xcodeproj for tests?
I guess SPM should cover this...

Reply:
Because we need to also test against iOS, tvOS and watchOS - not only macOS. So `xcodebuild` is still necessary. And since we keep all our settings neatly in xcconfig files, this is much easier to maintain with an Xcode Project. SPM tends to reset those settings and add its own.